### PR TITLE
1076-l10nScreenshotTests-delete

### LINKED ISSUE #1076: [l10nScreensthosTests] Add new tests for the Delete feature
This issue is to add the necessary tests to check the new strings added for the Delete feature.

### PR DESCRIPTION
**Pull request by isabelrios**
Fixes #1076
This PR adds a new test to check the screenshots related to the new Delete feature.

I had to run `fastlane snapshot update` to update the `SnapshotHelper.swift`so that tests work. That's why that file is modified.

### CODE REVIEW

**Comment by isabelrios** [comment]
Tests failing due to the number of logins, but that will be fixed once PR #1077 lands

File: LockwiseXCUITests/SnapshotHelper.swift [hunk 1]
```diff
@@ -153,35 +153,41 @@ open class Snapshot: NSObject {
...
+
             let window = app.windows.firstMatch
             let screenshot = window.screenshot()
-            guard let simulator = ProcessInfo().environment["SIMULATOR_DEVICE_NAME"], let screenshotsDir = screenshotsDirectory else { return }
-            let path = screenshotsDir.appendingPathComponent("\(simulator)-\(name).png")
+            guard var simulator = ProcessInfo().environment["SIMULATOR_DEVICE_NAME"], let screenshotsDir = screenshotsDirectory else { return }
+            
             do {
+                // The simulator name contains "Clone X of " inside the screenshot file when running parallelized UI Tests on concurrent devices
+                let regex = try NSRegularExpression(pattern: "Clone [0-1]+ of ")
```
Comment (jhugman) [inline LockwiseXCUITests/SnapshotHelper.swift]: This should probably be:

```regex
Clone \\d+ of
```

Unless `X` is always a binary number (only 0 and 1).

File: LockwiseXCUITests/SnapshotsXCUITests.swift [hunk 2]
```diff
@@ -133,4 +133,16 @@ class SnapshotsXCUITests: BaseTestCase {
...
+        loginToEntryListView()
+        // Show the Delete option
+        app.tables.cells.staticTexts["amazon.com"].swipeLeft()
+        snapshot("DeleteButton" + CONTENT_SIZE)
+        // Tap on Delete button
+        app.tables.buttons.firstMatch.tap()
+        snapshot("DeleteAlertDialog" + CONTENT_SIZE)
+        // Cancel deleting the login
+        app.alerts.buttons.firstMatch.tap()
+    }
```
Comment (jhugman) [inline LockwiseXCUITests/SnapshotsXCUITests.swift]: Ah. This is the bit doing the actual work of the bug.

**Comment by isabelrios** [comment]
@kaylagalway This PR is updated, please take a look and let me know if it is still approved or you want me to change anything. 
This is the [new set of screenshots](https://storage.googleapis.com/l10n-lockwise-ios/2019-11-21/screenshots/screenshots.html) taken, the Delete button and Delete alert dialog are there already :) As you can that later one is not translated, do you want me to open an issue for that or is that already known? 
Also, are we missing screenshots in other languages? In that case I can easily add the language to the `Snapfile` and take a new set of screenshots.
Thanks!

**Comment by kaylagalway** [comment]
@isabelrios yes, the translation issue is known. The translations for the delete work should be in the next release. 

As far as the screenshots in other languages thing - I'm not sure. Can check into that next week.

**Pull request merged**